pwa/src/pages/users/[id].tsx

```diff
@@ -1,16 +1,13 @@
 import * as React from "react";
```

Comment:
This is just an empty file 😂 

I don't think we have anything built around users. Might be worth to just remove this entire folder altogether.

However, I do understand that that's probably out of scope for this pull request. Maybe you can create a JIRA issue for this?